dbdimp.c

```diff
@@ -1930,12 +1934,31 @@ MYSQL *mysql_dr_connect(
 		*svp)
 	      cipher = SvPV(*svp, lna);
 
+	    if ((svp = hv_fetch(hv, "mysql_ssl_mode", 14, FALSE)) &&
+		*svp)
+            {
+              ssl_mode_str = SvPV(*svp, lna);
```

Comment:
ssl_mode_str is declared only when #if MYSQL_VERSION_ID >= SSL_MODE_VERSION

So this code needs to be #if-ed too.